main.lua

```diff
@@ -1,10 +1,14 @@
 Platform = require 'platform'
+Controller = require 'controller'
 
 platforms = {}
 
 function love.load()
 	love.graphics.setDefaultFilter('nearest', 'nearest')
 	love.graphics.setBackgroundColor(40, 50, 60)
+	local joysticks = love.joystick.getJoysticks()
```

Comment:
could pass `love.joystick.getJoysticks()` straight to the constructor if you wanted